tox.ini

```diff
@@ -8,6 +8,7 @@ deps =
     django111: Django>=1.11,<2.0
     django20: Django>=2.0,<2.1
     -rtravis.txt
+passenv = DATABASE_URL
```

Comment:
By default, [tox only passes the `PATH` environment variable](https://tox.readthedocs.io/en/stable/example/basic.html#passing-down-environment-variables). This is necessary to get the value for `DATABASE_URL` defined in .travis.yml passed to tox.